### PR TITLE
[application] ltfschoen

### PR DESCRIPTION
**Pull request by ltfschoen**

### CODE REVIEW

File: join_request/ltfschoen.yml [hunk 1]
```diff
@@ -0,0 +1,492 @@
+collective: Potoc
+member:
+  name: Luke Schoen
+  address: 1dsrQjL34njJ4Y8FXGyxeLnmunPZ6XAvid9jSQe9S4pTUh2
+  github: ltfschoen
```
Comment (wirednkod) [inline join_request/ltfschoen.yml]: please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 1dsrQjL34njJ4Y8FXGyxeLnmunPZ6XAvid9jSQe9S4pTUh2
  matrix:
  github: ltfschoen
```

Comment (ltfschoen) [reply join_request/ltfschoen.yml]: i've added the matrix handle now

**Comment by ltfschoen** [comment]
@wirednkod i've added another repo validator-scripts https://github.com/polkadot-tooling-collective/collective/pull/6/commits/3838e9ca8956f43e699be51b218283fda40d4ab9

**Comment by ltfschoen** [comment]
i've migrated relevant tooling code from https://github.com/ltfschoen/nft-coretime and https://github.com/ltfschoen/PolkadotJSTest to https://github.com/ltfschoen/open-polkadot-js-library and updated the collective file here

**Comment by ltfschoen** [comment]
* Add the following:
  * https://github.com/ltfschoen/pop-cli/pull/2
  * https://github.com/wuminzhe/scale_rb/pull/4

**Review by ggwpez — APPROVED** [review]
Accepted for work on relevant projects (PAPI, PJS, PJS-common, subxt).

**Pull request merged**